### DISCUSSION TITLE
Could not run source riddler: unexpected status code 500

### DISCUSSION THREAD

**Discussion by kschluns** [projectdiscovery/subfinder#805] [Q&A]
I'm noticing that Subfinder is often failing to run the Riddler source with status code 500 in the error message. Has anyone else had issues with Riddler in the past and is there anything one can do to reduce the likelihood of 500 error? 

EDIT: It's a status 403 error not a 500 error.

**Comment by olearycrew** [comment]
I can't even get Riddler's website to load - I wonder if they are having some sort of server issues?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
